audit/projects/k8s-staging-cluster-api-gcp/services/enabled.txt

```diff
@@ -4,7 +4,6 @@ cloudkms.googleapis.com           Cloud Key Management Service (KMS) API
 compute.googleapis.com            Compute Engine API
 containeranalysis.googleapis.com  Container Analysis API
 containerregistry.googleapis.com  Container Registry API
-containerscanning.googleapis.com  Container Scanning API
```

Comment:
Expected, but unfortunately not from landed code yet.  This is me using cluster-api-gcp as a test case while iterating on https://github.com/kubernetes/k8s.io/pull/2016